CONTRIBUTING.md

```diff
@@ -2,29 +2,43 @@
 
 :tada::balloon::cake: **Welcome to the Turing Way repository!** :cake::balloon::tada:
 
-:dizzy::hatched_chick::sunny: *We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
+:dizzy::hatched*chick::sunny: \_We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
 
 The point of this guide is to **welcome new users and contributors to the Turing Way community**.
 We hope that these guidelines are designed to make it as easy as possible to get involved.
 Don't let trying to be perfect get in the way of being good - we welcome all contributions and would love it if you could follow these guidelines to make sure your contributions can be easily integrated but exciting ideas are more important than perfect pull requests. :heart:
 
-
 If you have any questions that aren't discussed below, please let us know through one of the many ways to [get in touch](#get-in-touch).
 
 ## Table of contents
 
 Been here before? Already know what you're looking for in this guide? Jump to the following sections:
 
-* [Joining the community](#joining-the-community)
-* [Inclusivity](#inclusivity)
-* [Get in touch](#get-in-touch)
-* [Contributing through GitHub](#contributing-through-github)
-* [Writing in Markdown](#writing-in-markdown)
-* [Where to start: issues](#where-to-start-issues)
-* [Making a change with a pull request](#making-a-change-with-a-pull-request)
-* [The process of writing chapters](#the-process-of-writing-chapters)
-* [Style Guide](#style-guide)
-* [Recognising Contributions](#recognising-contributions)
+- [Contributing to the Turing Way](#contributing-to-the-turing-way)
+  - [Table of contents](#table-of-contents)
+  - [Joining the community](#joining-the-community)
+  - [Inclusivity](#inclusivity)
+  - [Get in touch](#get-in-touch)
+  - [Contributing through GitHub](#contributing-through-github)
+  - [Writing in Markdown](#writing-in-markdown)
+  - [Where to start: issues](#where-to-start-issues)
+    - [Issue labels](#issue-labels)
+  - [Making a change with a pull request](#making-a-change-with-a-pull-request)
+    - [1. Comment on an existing issue or open a new issue referencing your addition](#1-comment-on-an-existing-issue-or-open-a-new-issue-referencing-your-addition)
+    - [2. [Fork][github-fork] the [Turing Way repository][turing-way-repo] to your profile](#2-forkgithub-fork-the-turing-way-repositoryturing-way-repo-to-your-profile)
```

Comment:
ToC link broken - perhaps removing links within header will fix it?

Reply:
Ha, this looks like nested links... Inception style!

Reply:
This is an issue with the "Markdown All in One" extension on VS Code. The Prettier extension works fine.